src/assets/styles/includes.scss

```diff
@@ -17,7 +17,8 @@
 @mixin focus-style-base() {
   outline-offset: 0;
   outline-color: transparent;
-  transition: outline-offset 100ms ease-in-out, outline-color 100ms ease-in-out;
+  transition: inherit, outline-offset 100ms ease-in-out,
```

Comment:
oh, nice, does this allow the focus styles to not clobber an element's other transitions? Very clever.

Reply:
You know, that's what I was hoping it was going to do... but it doesn't, I'll remove it 🙈. We'd need to concatenate a passed transition in the mixin somehow to actually achieve that ☹️

Reply:
darn I thought you cracked the code

Reply:
yeah I guess focus style base would just accept a transition string. Little weird but would work.